src/test/java/org/opentripplanner/generate/doc/framework/DocsTestConstants.java

```diff
@@ -14,7 +14,7 @@ public interface DocsTestConstants {
   File USER_DOC_PATH = new File(DOC_ROOT, "user");
 
   /**
-   * This method return {@code true} if the /docs directory is available. If not, a warning is
+   * This method return {@code true} if the /doc directory is available. If not, a warning is
```

Comment:
This should probably say `/doc/user` right? At least that is the path that the code is looking at.